### PR TITLE
[NCL-5613] add kafka logging

### PR DESCRIPTION
**Pull request by michalovjan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -97,6 +97,7 @@
     <version.org.jboss.pnc.build.finder>2.0.1-SNAPSHOT</version.org.jboss.pnc.build.finder>
     <version.org.jboss.pnc.pnc-api>2.2.0</version.org.jboss.pnc.pnc-api>
     <version.org.jboss.resteasy>5.0.1.Final</version.org.jboss.resteasy>
+    <version.pnc.quarkus-kafka-logging>1.0.0-SNAPSHOT</version.pnc.quarkus-kafka-logging>
```
Comment (pkocandr) [inline pom.xml:100]: 1.0.0 was already released, see https://repo1.maven.org/maven2/org/jboss/pnc/logging/quarkus-logging-kafka/1.0.0/
No need for a snapshot.

Comment (pkocandr) [reply pom.xml:100]: Also would be probably better to use quarkus-logging-kafka, not quarkus-kafka-logging, to match the project name.

**Review by pkocandr — CHANGES_REQUESTED** [review]
Please update the version and the property name, otherwise it looks fine.

File: pom.xml (hunk 1 above)
Comment (michalovjan) [reply pom.xml:100]: yes sir!

**Review by pkocandr — APPROVED** [review]
Looks good.

**Pull request merged**